app/styles/ember-arcgis-portal-components.scss

```diff
@@ -214,7 +214,7 @@ div.panel-body:has(.item-picker) {
       flex-direction: column;
       overflow: auto;
       height: 100%;
-      justify-content: space-between;
+      margin: -10px 0;
```

Comment:
Why the negative margin? In general I think negative margins are a code smell. Sometimes they are needed but that is rare.

Reply:
This was following the Mozilla guide. I can remove it and see what happens.